### ISSUE TITLE
Add alternate text for annotate icon in sidebar

### ISSUE THREAD

**Issue by mattdricker** [hypothesis/support#241]
When there are no annotations to display, the Hypothesis sidebar shows a box with the following text:
`There are no annotations in this group. Create one by selecting some text and clicking the [annotate icon] button.`
where **[annotate icon]** is an SVG image shown on the button to create an annotation

This image does not have any corresponding alternate text or markup for parsing by screen readers or other accessibility tools.
<img width="468" alt="Screen Shot 2021-10-15 at 4 17 19 PM" src="https://user-images.githubusercontent.com/71603629/137549623-ebae4336-75fc-47f3-9f02-b907242e9f70.png">

**Comment by mattdricker** [comment]
The following resource may be helpful for SVG accessibility options: https://www.deque.com/blog/creating-accessible-svgs/
